packages/smooth_app/lib/pages/onboarding/welcome_page.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:smooth_app/pages/onboarding/country_selector.dart';
+import 'package:smooth_app/pages/page_manager.dart';
+import 'package:smooth_app/themes/smooth_theme.dart';
+import 'package:smooth_ui_library/util/ui_helpers.dart';
+
+/// Welcome page for first time users.
+class WelcomePage extends StatelessWidget {
```

Comment:
Should probably be named `WelcomePage1` as several "next" welcome pages are expected.

Reply:
Nope, this is "WelcomePage", next one I'm calling "ScanExplanation" or "ScanExample" and next one something else